cmd/podman/restart.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/urfave/cli"
+)
+
+var (
+	restartFlags = []cli.Flag{
+		cli.UintFlag{
+			Name:  "timeout, time, t",
+			Usage: "Seconds to wait for stop before killing the container",
+			Value: libpod.CtrRemoveTimeout,
+		},
+		LatestFlag,
+	}
+	restartDescription = `Restarts one or more running containers. The container ID or name can be used. A timeout before forcibly stopping can be set, but defaults to 10 seconds`
+
+	restartCommand = cli.Command{
+		Name:                   "restart",
+		Usage:                  "Restart one or more containers",
+		Description:            restartDescription,
+		Flags:                  restartFlags,
+		Action:                 restartCmd,
+		ArgsUsage:              "CONTAINER [CONTAINER ...]",
+		UseShortOptionHandling: true,
+	}
+)
+
+func restartCmd(c *cli.Context) error {
+	args := c.Args()
+	if len(args) < 1 && !c.Bool("latest") {
+		return errors.Wrapf(libpod.ErrInvalidArg, "you must provide at least one container name or ID")
+	}
+
+	if err := validateFlags(c, restartFlags); err != nil {
+		return err
+	}
+
+	runtime, err := getRuntime(c)
+	if err != nil {
+		return errors.Wrapf(err, "error creating libpod runtime")
+	}
+	defer runtime.Shutdown(false)
+
+	var lastError error
+
+	timeout := c.Uint("timeout")
```

Comment:
Can't we just set timeout to -1, and then allow user to specify the timeout. No need for special boolean.   Or just set it to 0, and make users specify timeout > 0.